ocfweb/docs/docs/staff/policy.md

```diff
@@ -0,0 +1,83 @@
+[[!meta title="Staff policy"]]
+
+
+Staff members shall be chosen by the OCF Decision Making Process. [per OCF
+Board decision on 4/13/89]
+
+
```

Comment:
Let's keep in the comment about the rest of the policy being established by the SM